src/locales/en.json

```diff
@@ -130,7 +130,8 @@
     "show-institution-details": "Show Institution Details",
     "result": {
       "name": "Panel Results",
-      "title": "Total Genes Evaluated"
+      "title": "Total Genes Evaluated",
```

Comment:
Change to "Genes Evaluated (Symbol & Synonyms)"

Reply:
done